src/sentry/api/endpoints/organization_sdk_updates.py

```diff
@@ -63,6 +65,10 @@ def get(self, request, organization):
 
         project_ids = self.get_requested_project_ids(request)
         projects = self.get_projects(request, organization, project_ids)
+
+        len_projects = len(project_ids)
```

Comment:
can we add the ungrouped tag here too?

Reply:
I actually was intentionally limiting the number of tags, I don't usually use ungrouped. Do you think it'd be helpful?

Reply:
Yeah ungrouped has been helpful in debugging before, one more tag shouldn't be a huge deal and imo better to have it and not use it than want it and not have it 🤷

Reply:
yeah np np